plugins/database/influxdb/connection_producer.go

```diff
@@ -192,6 +192,8 @@ func (i *influxdbConnectionProducer) createClient() (influx.Client, error) {
 			if err != nil || tlsConfig == nil {
 				return nil, errwrap.Wrapf(fmt.Sprintf("failed to get TLS configuration: tlsConfig:%#v err:{{err}}", tlsConfig), err)
 			}
+		} else {
+			tlsConfig = &tls.Config{}
```

Comment:
Instead of an else block what about changing line 171 to: `tlsConfig := &tls.Config{}` ?

Reply:
thanks, I have updated the pr